python/packages/autogen-agentchat/src/autogen_agentchat/agents/_assistant_agent.py

```diff
@@ -66,7 +66,7 @@ class AssistantAgentConfig(BaseModel):
     name: str
     model_client: ComponentModel
     tools: List[ComponentModel] | None = None
-    workbench: ComponentModel | None = None
+    workbenches: List[ComponentModel] | None = None
```

Comment:
minor tiny nit. But workbench probably doesnt need to change to workbenches. 
Workbench can imply plural .. multiple things in it.

Reply:
This is also a breaking change, can we leave the name as is and support a single arg or a list

Reply:
This change souldn't be breaking.
`workbench: Workbench | List[Workbench] | None = None,`
workbench is still input like this then converted to a list internally if a single workbench is input